inspire/base/templates/header.html

```diff
@@ -31,28 +28,169 @@
 ## structure: (endpoint, values, blueprint, caption)
 #}
 {% set navigation_bar = [
-  ('search.index', {}, 'search', 'Home'),
-  ('webdeposit.create', {"deposition_type": "literature"}, 'webdeposit', 'Submit'),
-  ('/inspire/help', {}, '', 'Help'),
+('search.index', {}, 'search', 'Home'),
+('webdeposit.create', {"deposition_type": "literature"}, 'webdeposit', 'Submit'),
+('/inspire/help', {}, '', 'Help'),
 ] -%}
 
-<section>
-  <nav id="topnav" class="navbar navbar-fixed-top navbar-inverse navbar-custom" role="navigation">
-    <div class="container">
-      <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-ex1-collapse">
-          <span class="sr-only">{{ _("Toggle navigation") }}</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
-        <a class="navbar-brand" href="{{ url_for('search.index') }}">
-          <img src="{{ url_for('static', filename='img/labs_white.svg') }}" alt="{{ config.CFG_SITE_NAME_INTL['en'] }}">
-        </a>
-      </div>
-
-      <div class="navbar-collapse navbar-ex1-collapse collapse">
-        <ul class="nav navbar-nav navbar-right">
+
+<style>
```

Comment:
I think these are leftovers from the other work you were doing on the megadropdown? It should be removed from this PR.

And there should not be ``<style>`` tags. All styling should go in the appropriate css/less files.